ext_localconf.php

```diff
@@ -382,6 +383,18 @@
     ]
 );
 
+\TYPO3\CMS\Extbase\Utility\ExtensionUtility::configurePlugin(
+    'Dlf',
+    'ValidationForm',
+    [
+        \Kitodo\Dlf\Controller\ValidationFormController::class => 'main',
```

Comment:
Shouldn't the validation form be non-cacheable?

Reply:
The plugin provides a container for requesting the validation middleware and displaying the validation response. This is primarily handled by JavaScript on the client side. The plugin's form itself can currently be cached, or rather I have not encountered any caching issues so far. If the plugin configuration is changed while being used as a content component, the page must be saved, which will automatically clear the page cache.

Reply:
Thanks for clarification!